src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Ingest metrics into the Observability Platform
+linkTitle: Metrics
+description: How to ingest new metrics in the Observability Platform
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
+    identifier: tutorials-observability-data-ingestion-metrics
+weight: 50
+last_review_date: 2024-07-11
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - What is a ServiceMonitor?
+  - What is a PodMonitor?
+  - How do I ingest new metrics?
+  - Why do my clusters run the Prometheus Operator?
+---
+
+By default, all Giant Swarm clusters are equiped with the [Prometheus Operator](https://prometheus-operator.dev/) and a set monitoring agents (multiple [Prometheus](https://prometheus.io/) shards in `agent mode`) to be able to collect and forward cluster critical and workload metrics to a [Grafana Mimir](https://grafana.com/oss/mimir/) instance running on the management cluster.
+
+As no workload is the same, especially in the way it exposes its metrics, we decided to base our monitoring configuration on the `ServiceMonitor` and `PodMonitor` CRDs provided by the Prometheus Operator. Those allow us to:
```

Comment:
```suggestion
No workload is the same, especially in the way it exposes its metrics, so the Observability Platforms monitoring configuration needs to be flexible. That's why it is based on the `ServiceMonitor` and `PodMonitor` Custom Resource Definitions (CRDs) provided by the Prometheus Operator. 

Those allow you to:
```

*   First use of CRD we should use the full name and add the abbrevation in brackets 
*   In docs customers don't need to know our perspective. Think about it, docs are somewhat easier to read if they take on your perspective as a reader, because you're interested in understanding and achieving something, not in emphasizing with the product team :)